test/data/test.css

```diff
@@ -130,7 +130,7 @@ h6 {
     content: "SHOULD NOT BE INCLUDED";
 }
 
-@media all and (min-width: 1024px) {
```

Comment:
Not sure why you removed these

Reply:
To pass the tests :-)
When puppeteer/chome collect css from a page it ignore "all and" 
and then tests failed.